subiquitycore/core.py

```diff
@@ -32,13 +32,15 @@ class Application:
     # A concrete subclass must set project and controllers attributes, e.g.:
     #
     # project = "subiquity"
-    # controllers = [
+    # controllers = {
+    #    "server": [
```

Comment:
Please remove all this file diff, not correct anymore with the current implementation (this was already a comment in the previous review)